pyspades/server.py

```diff
@@ -214,27 +220,49 @@ def get_cp_entities(self):
             entities.append(flag)
         return entities
 
-    def update(self):
-        self.loop_count += 1
+    async def update(self):
+        while True:
+            start_time = time.monotonic()
+            lag = start_time - self.world_time - UPDATE_FREQUENCY
+            if lag > 0.004:
+                log.debug("LAG before world update: " + str(round(lag*1000)) + " ms")
```

Comment:
```suggestion
                log.debug("LAG before world update: {lag:.1f}ms", lag=lag * 1000)
```

(Otherwise the string gets calculated every time, even in debug mode)